pom.xml

```diff
@@ -474,7 +474,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.22.1</version>
+        <version>2.22.2</version>
```

Comment:
I increased the version cause it's more stable unlike 2.22.1 and I can't get coverage info and successful pre-merge build with 2.22.1